packages/at_persistence_secondary_server/lib/src/model/at_metadata_builder.dart

```diff
@@ -61,58 +40,35 @@ class AtMetadataBuilder {
         ? atMetaData.version = 0
         : atMetaData.version = (existingMetaData!.version! + 1);
 
```

Comment:
Almost all of the metadata merging which used to happen here now happens in the UpdateVerbHandler. This is very important, since only the UpdateVerbHandler knows whether a field has deliberately been set to null by the client where it previously was non-null.

The only metadata merging that is required here is stuff that is very core to the persistence - i.e. createdBy / updatedBy, createdAt / updatedAt, and there is an immutable check here for safety's sake.